packages/core-contracts/contracts/interfaces/IBeacon.sol

```diff
@@ -1,8 +0,0 @@
-//SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
-
-interface IBeacon {
-    function getImplementation() external view returns (address);
-
-    function upgradeTo(address newImplementation) external;
-}
```

Comment:
How is this related to satellites? I think beacons are still useful and we might use at some point. However, I do agree to remove anything not being used atm. So your call...